distribution/docker-compose.squid-cache.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
```

Comment:
My understanding is that this is a compose file override file which will be incorporated in `tests/utilities` for test set up?

If so, I recommend adding some helpfully explanatory comments to this file to make that relationship clear in a follow up set of changes.

Reply:
I named it as clearly as I could, but right. Doing.

Reply:
done